jsapp/js/components/processing/sidebar/sidebarDisplaySettings.module.scss

```diff
@@ -0,0 +1,46 @@
+@use '~kobo-common/src/styles/colors';
+@use 'scss/sizes';
+
+.root {
+  display: flex;
+  justify-content: right;
+  padding-top: sizes.$x18;
+  padding-bottom: sizes.$x10;
+}
+
+.description {
+  padding-bottom: sizes.$x30;
+}
+
+.display {
+  padding-bottom: sizes.$x40;
+}
+
+.hasTranslation {
+  span::before {
+    content: '(';
+  }
+
+  span::after {
+    content: ')';
+  }
+}
+
+.options {
+  height: 100%;
+  padding-bottom: sizes.$x24;
+  overflow-y: scroll;
+}
+
+footer {
+  padding-top: sizes.$x12;
+  padding-bottom: sizes.$x12;
+  justify-content: center !important;
+  // Some basic overflow styling just in case the list is very long
+  position: fixed;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  background: colors.$kobo-white;
+  border-radius: sizes.$x6;
```

Comment:
All of these styles are actually not needed. The `kobo-modal__footer` element is always at the bottom, and the `kobo-modal__content` never overflows it.

Reply:
I think we should remove this whole `footer {…}` from here. Please in `koboModal.scss` file add `.kobo-modal__footer--centered` that simply has `justify-content: center;`. And then in tsx file use `<KoboModalFooter isCentered>`. And finally edit `koboModalFooter.tsx` and add the `isCentered` prop and `<bem.KoboModal__footer m={props.isCentered ? 'isCentered' : ''}>`.

Reply:
Note for potential cleanup: this needed `justify-content: center !important`